resources/quality/deltacomb/deltacomb_abs_fast.inst.cfg

```diff
@@ -0,0 +1,25 @@
+[general]
+version = 2
+definition = deltacomb
+name = Fast Quality (beta)
+
+[metadata]
+type = quality
+setting_version = 3
```

Comment:
All `setting_version` for the qualities should now be `4`. The rest look good.

Reply:
Thank you!